building/build-debs/homeworld-admin-tools/src/setup.py

```diff
@@ -212,6 +231,7 @@ def wrap_param_tx(params):
     "keyserver": wrapop("deploy keys and configuration for keyserver; start keyserver", setup_keyserver),
     "self-admit": wrapop("admit the keyserver into the cluster during bootstrapping", admit_keyserver),
     "keygateway": wrapop("deploy keytab and start keygateway", setup_keygateway),
+    "update-keytab": wrapop("update keytab and restart keygateway", update_keytab),
```

Comment:
These two commands should have clearer names. Also, keep in mind that the parameter to setup_keygateway will be treated as an optional command-line parameter, as far as I remember, which might not be what you want.

Reply:
What names do you suggest? "Set up keygateway" and "update the keytab" sounds like exactly what those commands are for. Would it be better to actually have force-overwrite as an optional string parameter, so that the administrator would call "spire setup keygateway --force-overwrite" to update the keytab?

Also thanks for noting that, will fix.

Reply:
I'd name it `update-keygateway`, so that it's clear that the operations are very similar.